pkg/cassandra/config/config.go

```diff
@@ -58,6 +58,24 @@ type Schema struct {
 	// while connecting to the Cassandra Cluster. This is useful for connecting to clusters, like Azure Cosmos DB,
 	// that do not support SnappyCompression.
 	DisableCompression bool `mapstructure:"disable_compression"`
+	//Datacenter is the name for network topology
+	Datacenter string `mapstructure:"datacenter" valid:"optional"`
+	// TraceTTL is Time To Live (TTL) for the trace data in seconds
+	TraceTTL int `mapstructure:"trace_ttl" valid:"optional"`
+	// DependenciesTTL is Time To Live (TTL) for dependencies data in seconds
+	DependenciesTTL int `mapstructure:"dependencies_ttl" valid:"optional"`
+	// Replication factor for the db
+	ReplicationFactor int `mapstructure:"replication_factor" valid:"optional"`
+	// CasVersion is version of cassandra used
+	CasVersion int `mapstructure:"cas_version" valid:"optional"`
+	// CompactionWindow of format "^[0-9]+[mhd]$" tells the compaction window of the db
+	CompactionWindow string `mapstructure:"compaction_window" valid:"optional"`
+	// Replication is the replication strategy used
+	Replication string `mapstructure:"replication" valid:"optional"`
+	// CompactionWindowSize is the numberical part of CompactionWindow. Extracted from CompactionWindow
+	CompactionWindowSize int `mapstructure:"compaction_window_size" valid:"optional"`
+	// CompactionWindowUnit is the time unit of the CompactionWindow. Extracted from CompactionWindow
+	CompactionWindowUnit string `mapstructure:"compaction_window_unit" valid:"optional"`
```

Comment:
if we use time.Duration type we will not need the unit separately

Reply:
Also, Size and Unit seems to be derived fields, if so they should not be declared in the Config struct. You can create a different struct for use with Go templates, e.g.
```go
type TemplateParams struct {
  config.Schema // embed config
  // add other derived fields
}
```

Reply:
Added TemplateParams